docs/releases.rst

```diff
@@ -1,6 +1,333 @@
 Release notes
 =============
 
+Mailu 2.0 - 2023-04-03
+----------------------
+
+Mailu 2.0 is finally available. It is vital to read the `Upgrading` section before upgrading to Mailu 2.0.
+
+Highlights
+``````````
+
+This is an overview of the major features introduced in Mailu 2.0.
+
+Multi-arch images (ARM support)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Mailu project now ships multi-arch images for the architectures:
+
+- linux/amd64
+- linux/arm64/v8
+- linux/arm/v7
+
+It is now possible to run Mailu on most ARM hardware such as the Raspberry Pi.
+
+Auto-configuration for client
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On the domain details page, there are new DNS records for enabling DNS auto-client configuration.
+Provided they are configured, email clients will make use of them to auto-configure.
+
+If a reverse proxy is in use, settings might have to be tweaked.
+
+For Apple users, the client setup page now offers an autoconfiguration link to automatically configure
+their device.
+
+RESTFul API
+^^^^^^^^^^^
+
+Mailu offers a RESTful API for changing the Mailu configuration.
+Now, anything that can be configured via the Mailu web administration interface
+can also be configured via the Mailu RESTful API.
+
+Configuring a new domain or add new users can be fully automated now.
+
+The current API makes use of a single API token for authentication.
+In a future release this will likely be re-visited.
+
+For more information refer to the `Mailu RESTful API` page.
+
+Header authentication support (use external identity providers)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is now possible to use different authentication systems (such as keycloak, authentik, vouch-proxy) to handle the authentication of Mailu users.
+This can be used to enable Single Sign On from other IDentity Providers via protocols such as OIDC or SAML2.
+
+For more information see `Header authentication using an external proxy` in the configuration reference.
+
+Better anti-spoofing protection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously Mailu would reject emails where an attacker spoofs the envelope-From. Now Mailu also checks the header-From for any hosted domain.
+It won't let any email which pretends to be for any of the local domains through unless they pass DMARC.
+
+Implement a password policy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In line with security best practices, we have introduced password policy.
+
+Passwords now need to:
+- be at least 8 characters long
```

Comment:
8 chars without MFA is not that much. That being said, it's ok as a default value but and admin would expect to be able to bump it up to 10 or 12 (cf. OWASP ASVS: https://github.com/OWASP/ASVS/blob/master/4.0/en/0x11-V2-Authentication.md#v21-password-security)
Is there a possibility to change that default value to something different? This should be highlighted here as well if it is (or not) possible to change it.

Reply:
We follow the NIST guideline.

"not that much" for what? In which threat model does it matter? We have controls against both offline bruteforce attempts (bcrypt with high-ish cost) and online bruteforce attempts (rate-limits).

I have clarified in 83ce32f7